src/sso/interfaces/connection.interface.ts

```diff
@@ -9,7 +9,7 @@ export interface ConnectionDomain {
 export interface Connection {
   object: 'connection';
   id: string;
-  status: 'linked' | 'unlinked';
+  state: 'linked' | 'unlinked';
```

Comment:
@logangingerich The values here aren't correct. They should be `state: 'draft' | 'active' | 'inactive'`.

Reply:
Ah, the Linear ticket didn't have that context. I'll update.

Reply:
We should also go through a migratory period where we support both in tandem temporarily, similar to what I mentioned here: https://github.com/workos-inc/workos-dotnet/pull/37#discussion_r626838549